.github/workflows/web.yml

```diff
@@ -71,8 +72,27 @@ jobs:
               - '**/tests/clapi_export/**'
               - '**/www/!(front_src)/**'
               - '**/doc/API/**'
+            has_features_changes:
+              - '**/tests/e2e/**/*.feature'
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
 
+  gherkin-lint:
+    runs-on: ubuntu-22.04
+    needs: [changes]
+    if: ${{ needs.changes.outputs.has_features_changes == 'true' }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Navigate to Centreon E2E Tests Directory
+        run: cd centreon/tests/e2e
+
+      - name: Install Gherkin Lint
+        run: npm install gherkin-lint@4.2.2
+
+      - name: Run Gherkin Lint
+        run: ./node_modules/.bin/gherkin-lint -c centreon/tests/e2e/.gherkin-lintrc centreon/tests/e2e/features
```

Comment:
To mâle this action as portable as possible, Can you create .gherkin-lintrc directly from Bash (with EOF)
You have an example in yaml-lint workflow